apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-editor.tsx

```diff
@@ -44,7 +48,9 @@ export const InAppEditor = ({ uiSchema }: { uiSchema?: UiSchema }) => {
           {body && (
             <>
               {getComponentByType({ component: body.component })}
-              <FormMessage>{`Type {{ for variables, or wrap text in ** for bold.`}</FormMessage>
+              <FormMessagePure
```

Comment:
Why is this necessary? I'd expect `<FormMessage />` to take care of this.

Reply:
because it's outside of the `FormField`

Reply:
Wont it be possible to add it to the `in-app-body` component directly?

Reply:
It was there, but Dima updated the code and reused the `getComponentByType,` which shouldn't be actually used for onboarding purposes. Maybe I'll take a look into that part.

Reply:
The error message belongs in the `InAppEditor`.